Index: app/src/main/java/pt/ulusofona/deisi/a2020/cm/g6/NavigationManager.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package pt.ulusofona.deisi.a2020.cm.g6\r\n\r\nimport androidx.fragment.app.Fragment\r\nimport androidx.fragment.app.FragmentManager\r\n\r\nabstract class NavigationManager {\r\n\r\n    companion object{\r\n        private fun placeFragment(fm: FragmentManager, fragment: Fragment){\r\n            val transition = fm.beginTransaction()\r\n            transition.replace(R.id.frame, fragment)\r\n            transition.addToBackStack(null)\r\n            transition.commit()\r\n        }\r\n\r\n        fun goToDashboardFragment(fm: FragmentManager){\r\n            placeFragment(fm, DashboardFragment())\r\n        }\r\n\r\n        fun goToRegisto(fm: FragmentManager){\r\n            placeFragment(fm, RegistoFragment())\r\n        }\r\n\r\n        fun goToListTestes(fm: FragmentManager){\r\n            placeFragment(fm, ListaTestesFragment())\r\n        }\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/pt/ulusofona/deisi/a2020/cm/g6/NavigationManager.kt	(revision c9e4c743ee9c6ecf2f523d0e20bfabb149b35f49)
+++ app/src/main/java/pt/ulusofona/deisi/a2020/cm/g6/NavigationManager.kt	(date 1617126354703)
@@ -24,5 +24,11 @@
         fun goToListTestes(fm: FragmentManager){
             placeFragment(fm, ListaTestesFragment())
         }
+
+        fun goToTest(fm: FragmentManager,test: TesteCovid){
+            placeFragment(fm, ())
+        }
+
+
     }
 }
\ No newline at end of file
